dp/cloud/python/magma/test_runner/tests/test_dp_with_orc8r.py

```diff
@@ -421,7 +442,7 @@ def build_post_data(self) -> Dict[str, Any]:
         }
 
     def build_post_data_with_serial_number(self, serial_number) -> Dict[str, Any]:
```

Comment:
how about separating `build` from `with`, so that you can write e.g. `CbsdAPIBuilder.WithSerialNumber('some_serial').build_post_data()`